bgpd/bgp_route.c

```diff
@@ -10428,6 +10428,40 @@ static int bgp_table_stats(struct vty *vty, struct bgp *bgp, afi_t afi,
 			   safi_t safi, json_object *json);
 
 
+DEFUN (show_ip_bgp_statistics_all,
+       show_ip_bgp_statistics_all_cmd,
+       "show [ip] bgp [<view|vrf> VIEWVRFNAME] statistics-all [json]",
+       SHOW_STR
+       IP_STR
+       BGP_STR
+       BGP_INSTANCE_HELP_STR
+       "Display number of prefixes for all afi/safi\n"
+       JSON_STR)
+{
+	bool uj = use_json(argc, argv);
+	json_object *json = NULL;
+	struct bgp *bgp = NULL;
+	safi_t safi;
+	afi_t afi;
+
+	bgp_vty_find_and_parse_bgp(vty, argv, argc, &bgp, uj);
+	if (!bgp)
+		return CMD_SUCCESS;
+	if (uj)
+		json = json_object_new_object();
+	for (afi = AFI_IP; afi < AFI_MAX; ++afi) {
```

Comment:
This function does not produce valid json output.  { 'afi' : 'l2vpn', then a list of safi's underneath it. }.

Reply:
I need some analoguous example from BGP.
```{"ipv4" :  [ {"unicast", "stat1:, ..}{"vpn", "stat1:,..}]}```

Reply:
show bgp summary json?

Reply:
thanks. done.